pkgs/tools/nix/nixpkgs-manual-mmdoc-watch/default.nix

```
@@ -0,0 +1,19 @@
+{ writeScriptBin, python3, entr } :
+let
+  np-build = writeScriptBin "np-build" ''
+    nix build .#nixpkgs-manual-mmdoc -L
+  '';
+in
+writeScriptBin "nixpkgs-manual-mmdoc-watch" ''
+  killbg() {
+  for p in "''${pids[@]}" ; do
+  kill "$p";
+  done
+  }
+  trap killbg EXIT
+  pids=()
+  ${python3}/bin/python -m http.server --directory ./result &
+  pids+=($!)
+  trap exit SIGINT
+  while true; do find doc pkgs/tools/nix/nixpkgs-manual-mmdoc/ | ${entr}/bin/entr -cd ${np-build}/bin/np-build; done
```

I am not sure if this results in a busy loop. Can't we use something like inotify-tools?

---

```suggestion
  while inotifywait -e modify -e create doc pkgs/tools/nix/nixpkgs-manual-mmdoc; do nix-build . -A nixpkgs-manual-mmdoc; done`
```

More uses if needed
https://github.com/search?q=lang%3Ash+%2Fwhile.%2Binotify%2F&type=code